docs/blitz/details/yang.rst

```diff
@@ -111,6 +111,7 @@ subscription, or, you may expect the test to fail (referred to as a negative tes
       auto-validate:   # [true | false] automatically validate config messages
       negative-test:   # [true | false] expecting device to return an error
       pause:           # pause N seconds between each test (default: 0, no pause)
+      sequence:        # [true | false] return values and return sequence verified
```

Comment:
sorry, what this means? return both `values` and `sequence verified`? I don't even know what they are.

Reply:
I don't know what it looks like... maybe better to have more explanation in different section or in same section?

Reply:
@tahigash, please open another issue about providing more detail on _all_ the format parameters and assign it to me.  I will take care of it.

Reply:
requested issue is created. https://github.com/CiscoTestAutomation/genie/issues/75
But I couldn't assign to you. so sent an invitation as outside collaborator. so that I think we can assign the issue to you. or please assign it by yourself. it would work as well.

I'm approving this PR now.